internal/business/server/healthcheck_handler.go

```
@@ -21,6 +21,41 @@ import (
+func maskResponse(r *healthcheck.Response) *healthcheck.Response {
+	copy := *r
+	copy.URL = "****"
+	if len(r.Errors) > 0 {
+		maskedErrors := make([]healthcheck.ErrorResponse, len(r.Errors))
+		for i, e := range r.Errors {
+			maskedErrors[i] = healthcheck.ErrorResponse{
+				Error:   e.Error,
+				Message: "****",
+			}
+		}
+		copy.Errors = maskedErrors
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)